content/html/concepts/elements/elements.md

```diff
@@ -61,6 +61,6 @@ HTML is organized into a family tree structure. HTML elements can have parents,
 
 So here are the relationships for `<h1>`:
 
-- `<h1>` is the child of `<div>`
-- `<h1>` is the grandchild of `<body>`
+- `<h1>` is the child of `<div>` and the `div` is the parent of the `<h1>`
+- `<h1>` is the grandchild of `<body>` and the `<body>` is the grandparent of the `<h1>`
```

Comment:
```suggestion
- `<h1>` is the child of `<div>`
- `<div>` is the parent of the `<h1>`
- `<h1>` is the grandchild of `<body>`
- `<body>` is the grandparent of the `<h1>`
```

Nice additions! I thought it'd be good to separate the mentions of the different relationships. I'm also good with the condensed version if you want to keep it more compact.